LivingCity/traffic/b18TrafficLaneMap.cpp

```diff
@@ -64,7 +67,15 @@ void B18TrafficLaneMap::createLaneMapSP(const std::shared_ptr<abm::Graph>& graph
   
   edgeDescToLaneMapNumSP.clear();
   laneMapNumToEdgeDescSP.clear();
+  
 
+  abm::graph::edge_id_t max_edge_id = 0;
+  for(auto it = graph_->edge_ids_.begin(); it != graph_->edge_ids_.end(); ++it){
```

Comment:
It might be more clear / idiomatic to iterate with the C++11 style foeach:

```suggestion
  for(const std::pair<std::tuple<graph::vertex_t, graph::vertex_t>, graph::edge_id_t : graph_->edge_ids_) {
```